.github/workflows/dispatch-rebottle.yml

```diff
@@ -29,27 +29,54 @@ env:
   HOMEBREW_RELOCATE_RPATHS: 1
 
 jobs:
+  setup:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/homebrew/ubuntu16.04:master
+    outputs:
+      runners: ${{steps.determine-runners.outputs.runners}}
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Determine runners
+        id: determine-runners
+        env:
+          HOMEBREW_FORMULA_TO_REBOTTLE: ${{github.event.inputs.formula}}
+          HOMEBREW_REBOTTLE_TIMEOUT: ${{github.event.inputs.timeout}}
```

Comment:
You can now pass these as arguments to `brew ruby` I think. Not that it makes a huge difference, but it probably makes the script easier to test locally.